hs_discover/src/components/Search.vue

```diff
@@ -1,29 +1,632 @@
 <template>
-    <div id="discover-search">
```

Comment:
Resources consolidated and renamed to Search cleaner and flatter. This was leftover from previous version that warranted this approach and is no longer the case.